homeassistant/components/powerwall/__init__.py

```diff
@@ -32,11 +30,6 @@
     UPDATE_INTERVAL,
 )
 
-CONFIG_SCHEMA = vol.Schema(
```

Comment:
We should keep it around, so users that still have such an entry are informed about it in the logs.

```py
CONFIG_SCHEMA = cv.deprecated(DOMAIN)
```